src/main/groovy/com/ullink/gradle/nunit/ReportGenerator.groovy

```diff
@@ -0,0 +1,116 @@
+package com.ullink.gradle.nunit
+
+import org.gradle.api.tasks.Exec
+
+class ReportGenerator extends Exec {
+
+    def ExtentReportType = 'v3html'
+    def ExtentReportDownloadUrl = 'https://github.com/extent-framework/extentreports-dotnet-cli/archive'
```

Comment:
it's better to download from https://www.nuget.org/api/v2/package/extent/0.0.3 so we can update the version easily

Reply:
Done